search-app/src/cmr/search/data/elastic_search_index.clj

```diff
@@ -194,6 +194,14 @@
                  "1_subscriptions")
    :type-name "subscription"})
 
+(doseq [concept-type (concepts/get-generic-concept-types-array)]
+  (defmethod common-esi/concept-type->index-info concept-type
+    [context _ query]
+    {:index-name (if (:all-revisions? query)
+                   (format "1_all_generic_%s_revisions" (name concept-type))
+                   (format "1_generic_%s" (name concept-type)))
```

Comment:
Are we able to move way from the `1_` prefix since CMR never implemented the index splitting as planned?

Reply:
we have a plan to move this to a configuration

Reply:
We looked into this, it's not easy with the existing code to give a different prefix for some indexes and not others. Right now the code packages all the indexes up as one set and added the same prefix to them. We would have to add code that allows for generics to have a different prefix. Also, it seams like to me that the version concept would allow us to create a new set of indexes in the future and maintain legacy support.

We think more thought should be given to this, and maybe create a ticket to do this work in the next PI.

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8620